src/main/resources/templates/tools.html

```diff
@@ -62,29 +61,52 @@ <h4 align="left"><strong>Ready-to-use Blockchain environment &amp; Algorithm val
             <ul>
                 <li>Secured Distributed ledger (Blockchain) is a distributed trust system whose security relies on a large number of peers.</li>
             </ul>
-            <p align="left"><strong>Potential Users</strong></p>
-            <ul>
-                <li>Financial community, government agencies and academia researchers/lecturers</li>
-            </ul>
+            <!--<p align="left"><strong>Potential Users</strong></p>-->
+            <!--<ul>-->
+                <!--<li>Financial community, government agencies and academia researchers/lecturers</li>-->
+            <!--</ul>-->
             <hr/>
         </div>
 
         <div class="col-sm-4 col-sm-offset-4">
             <hr/>
-            <h4 align="left"><strong>Ready-to-use Common Vulnerabilities and Exposure (CVE) Environments</strong></h4>
+            <h4 align="left"><strong>Virtual Mid-size Enterprise Network</strong></h4>
             <p align="left"><strong>What is this ?</strong></p>
             <ul>
                 <li>A medium-sized virtual network that can simulate multiple real life scenarios.</li>
                 <li>Useful for training and validation. </li>
             </ul>
-            <p align="left"><strong>Potential Users</strong></p>
-            <ul>
-                <li>Cyber exercise users, forensic training users, cyber security lecturers</li>
-            </ul>
+            <!--<p align="left"><strong>Potential Users</strong></p>-->
+            <!--<ul>-->
+                <!--<li>Cyber exercise users, forensic training users, cyber security lecturers</li>-->
+            <!--</ul>-->
             <hr/>
         </div>
-    </div>
-</div>
+
+            </div>
+        </div>
+        <section id="joinUs">
+            <div class="container ">
+                <div class="row ">
+                    <div class="heading text-center col-sm-12 background4 ">
+                        <div>
+                            <p class="h15">Data</p>
+                            <p>&nbsp;</p>
+                        </div></div>
+                    <div class="col-sm-12 ">
+                        <p>&nbsp;</p>
+                        <p>&nbsp;</p>
+                <div class="col-sm-1 col-sm-offset-1">
+                    <a href="http://impactcybertrust.org"> <img src="images/impact1.png" width="251" height="189" href="http://impactcybertrust.org"/></a>
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add an "alt" attribute to this image. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3AImgWithoutAltCheck)